src/uas/UASManagerInterface.h

```diff
@@ -98,6 +98,8 @@ public slots:
     void UASCreated(UASInterface* UAS);
     /** A system was deleted */
     void UASDeleted(UASInterface* UAS);
+    /** A system was deleted */
+    void UASDeleted(int systemId);
```

Comment:
Since its for an instance of the class rather than for a particular UAS ID we could change it, yes.